### PR TITLE
Add a mutable special netherum monster corpse "dungeon". Part 2: OMTs and Terrain

### PR DESCRIPTION
**Pull request by gkarfakis19**
#### Summary
Content "Adds a special buried netherum squid monster structure, part 2: OMTs and Terrain"

#### Purpose of change

According to the lore, during the massive portal storms that shook Earth once the Blob first attacked, just a few days before the start date, skyscraper-sized monsters ravaged civilization. This is a first attempt to realize this part of the lore by including a subterranean buried netherum squid monster corpse that spans several overmap tiles.
This is a large-ish project that will ideally include some special nether scavenger monsters that inhabit the corpse, as well as custom tiles to decorate the interior.
This part 2 PR will focus on the OMTs and specific terrain inside the monster corpse.

#### Describe the solution

Currently, I have began a preliminary process of populating the insides of the monster. The basic plan is:

- [x] Populate neck segments (about 90% done).
- [x] Implement the outline of a simple circulatory system of ichor that runs through the beast
- [x] Add some proof-of-concept custom denizens/furniture in the beast. Will be heavily expanded later, in a later PR.
- [ ] Populate tentacles, perhaps with some extra small organoids and squid suckers.
- [ ] Populate the interior of the bowels, a very cramped maze-like space.
- [ ] Populate the interior of the head, which should include a huge brain flanked by ichor veins

Make some sample tiles for Ultica to ensure everything looks correct (over at https://github.com/I-am-Erk/CDDA-Tilesets/pull/1168)
- [x] Misc tiles
- [x] Brain
- [ ] Organs

The currently added proof-of-concept furniture include some glowing nether spore "gas" emitters. The spores act like smoke if inhaled without a gas mask, but regardless of environmental protection status will always coat the player, causing them to glow and attracting unwanted attention. 
I have also added a host of nether flora, some with unique effects like the irresistible nether lily that just makes you want to sit and admire its pristine, luscious form.

Also added a small nether fish symbiote monster that acts like a blind piranha, being a medium difficulty combatant that is attracted to sound, adding an incentive for the player to be relatively stealthy to avoid needless combat with these monsters while crossing the numerous ichor veins inside the beast.

#### Testing

Heavily WIP, but here's a very preliminary screenshot of the neck area using existing textures (will look quite different once Ultica and the other tilesets start to support the custom monster tiles).
image

Similar image with some basic recolourings of the Ultica tileset that I quickly made:
image

### CODE REVIEW

File: doc/EXAMINE.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The examine actors are specified as JSON objects with a `type` corresponding to
 
 ## Hardcoded Examine Actions
 
+- ```admire``` Admires a furniture tile (flavour).
```
Comment (gkarfakis19) [inline doc/EXAMINE.md:9]:
```suggestion
- ```admire``` Admires a furniture tile (flavor).
```

**Comment by Maleclypse** [comment]
I'm really excited for this, do you need any help?

**Comment by gkarfakis19** [comment]
> I'm really excited for this, do you need any help?

I'm extremely busy at the moment and will continue to be for the next month or so, so I plan to continue work on this PR after. In any case, I'd welcome any help from an experienced contributor once I've got more free time, so thanks!

**Comment by MNG-cataclysm** [comment]
@Maleclypse You should probably move this to the Done section in the Abandoned But Desired PR's project, as all the content here has been added to the game and expounded upon.

**Comment by Maleclypse** [comment]
Thanks and done!

**Pull request closed without merging**